pkg/virt-operator/webhooks/kubevirt-update-admitter.go

```diff
@@ -117,6 +117,23 @@ func (admitter *KubeVirtUpdateAdmitter) Admit(ar *admissionv1.AdmissionReview) *
 		response.Warnings = append(response.Warnings, warnDeprecatedFeatureGates(featureGates, admitter.ClusterConfig)...)
 	}
 
+	const mdevWarningfmt = "%s is deprecated, use mediatedDeviceTypes"
+	if mdev := newKV.Spec.Configuration.MediatedDevicesConfiguration; mdev != nil {
+		f := field.NewPath("spec", "configuration", "mediatedDevicesConfiguration")
+		if mdev.MediatedDevicesTypes != nil || len(mdev.MediatedDeviceTypes) != 0 {
```

Comment:
I don't understand this condition... Is there a typo + a logic error? Seems unlikely but I would have expected something more like:
```suggestion
		if mdev.MediatedDevicesTypes != nil && len(mdev.MediatedDevicesTypes) != 0 {
```

Reply:
Actually, I think `len(mdev.MediatedDevicesTypes) != 0` would be enough. Why do we need `&&`?